src/BrowserPane.m

```diff
@@ -705,11 +694,12 @@ -(IBAction)handleReload:(id)sender
 -(void)handleStopLoading:(id)sender
 {
 	[self willChangeValueForKey:@"isLoading"];
+	[self.webPane.preferences setPlugInsEnabled:NO];
```

Comment:
I do not know why, but this seems to disable plugins globally. I could not load a video in any other tab after closing another one.

Reply:
Can you confirm ? I prefer not to reinstall Flash player, but I see no problem with the Adobe Acrobat plugin.

Reply:
Not sure whether it is adobe flash, I encounter this on the custom players of the german news page http://www.tagesschau.de . Possibly they are flash based.

Reply:
The Web inspector (enabled through Terminal `defaults write uk.co.opencommunity.vienna2 WebKitDeveloperExtras -bool true` and contextual menu) seems to indicate that tagesschau.de does not use Flash. 

Confirmed also by the fact that the problem is "fixed" if you go in the Advanced Preferences to disable/reenable Javascript.